sdk/src/wallet/account/types/mod.rs

```diff
@@ -114,7 +116,7 @@ pub struct OutputDataDto {
     /// Remainder
     pub remainder: bool,
     /// Bip32 path
-    pub chain: Option<Chain>,
+    pub chain: Option<Bip44>,
```

Comment:
This also needs `#[serde(with = "option_bip44")]`?

Reply:
Yeah, done